Makefile.pre.in

```diff
@@ -1205,6 +1205,18 @@ Modules/getpath.o: $(srcdir)/Modules/getpath.c Python/frozen_modules/getpath.h M
 		-DPLATLIBDIR='"$(PLATLIBDIR)"' \
 		-o $@ $(srcdir)/Modules/getpath.c
 
+# like getpath.o with additional -DPY_BOOTSTRAP_PYTHON=1
+Modules/getpath_bootstrap.o: $(srcdir)/Modules/getpath.c Python/frozen_modules/getpath.h Makefile $(PYTHON_HEADERS)
+	$(CC) -c $(PY_CORE_CFLAGS) -DPYTHONPATH='"$(PYTHONPATH)"' \
+		-DPREFIX='"$(prefix)"' \
+		-DEXEC_PREFIX='"$(exec_prefix)"' \
+		-DVERSION='"$(VERSION)"' \
+		-DVPATH='"$(VPATH)"' \
+		-DPLATLIBDIR='"$(PLATLIBDIR)"' \
+		-DPY_BOOTSTRAP_PYTHON=1 \
+		-o $@ $(srcdir)/Modules/getpath.c
```

Comment:
Wouldn't it be more elegant to have a two-line file named `getpath_bootstrap.c` using the default .c.o rule?
```c
#define BOOTSTRAP_PYTHON=1
#include "getpath.c"
```

Reply:
I agree that the approach is not elegant. ``getpath.c`` requires additional macros https://github.com/python/cpython/blob/ddbab69b6d44085564a9b5022b96b002a52b2f2b/Modules/getpath.c#L24-L28